lib/broccoli/bundler.js

```diff
@@ -148,8 +150,9 @@ module.exports = class Bundler {
     options = options || {};
 
     let scriptOutputFiles = options.scriptOutputFiles;
+    let applicationAndDepsDebugTree = this._debugTree(applicationAndDepsTree, 'bundler:js:input');
```

Comment:
The string passed to `BroccoliDebug.buildDebugCallback` is prefixed for all trees debugged by that particular `debugTree` callback. This will cause the debug dir to be called `DEBUG/bundler:bundler:js:input`.